pytorch-job/pytorch-operator/base/kustomization.yaml

```diff
@@ -12,4 +12,4 @@ commonLabels:
 images:
 - name: gcr.io/kubeflow-images-public/pytorch-operator
   newName: gcr.io/kubeflow-images-public/pytorch-operator
-  newTag: vmaster-gd596e904
+  newTag: vmaster-gd6623278
```

Comment:
Why after this commit: https://github.com/kubeflow/pytorch-operator/commit/d6623278ad47c97247d7d49f88f68c59adac5305, tags are updated: https://console.cloud.google.com/gcr/images/kubeflow-images-public/GLOBAL/pytorch-operator?gcrImageListsize=30 to v1.1.1, v0.6.0 and vmaster ?
/cc @jlewi @johnugeorge @gaocegege

Reply:
@andreyvelich That's how the apps-cd pipelines work. They attach a tag based on the config
https://github.com/kubeflow/testing/blob/637d1cd5fe33d03ee5646380d960fabe8a230d0a/apps-cd/applications.yaml#L234

Reply:
Does this CD attach tag after every commit on `master`, `v1.1-branch` and `v1.0-branch` branch ?